.github/workflows/ccpp.yml

```
@@ -18,10 +18,14 @@ jobs:
     - name: autoreconf
       run: autoreconf -i -f
     - name: configure
-      run: ./configure
+      run: ./configure --prefix=$PWD/_i
     - name: make
       run: make
     - name: make check
       run: make check
     - name: make distcheck
       run: make distcheck
+    - name: make install
+      run: make install
+    - name: ambs-lgp2-client
+      run: set -x; abs_top_builddir="$PWD"; export PKG_CONFIG_PATH="${abs_top_builddir}/_i/lib/pkgconfig"; export LD_LIBRARY_PATH="${abs_top_builddir}/_i/lib"; cd examples/ambs-lgp2-client && autoreconf -vis && ./configure --prefix="$PWD/_i" && make && ldd ambs-lgp2-client && ./ambs-lgp2-client && make install && ldd _i/bin/ambs-lgp2-client && _i/bin/ambs-lgp2-client
```

I'd suggest using multi line strings.

```suggestion
      run: |
        set -x
        abs_top_builddir="$PWD"
        export PKG_CONFIG_PATH="${abs_top_builddir}/_i/lib/pkgconfig"
        export LD_LIBRARY_PATH="${abs_top_builddir}/_i/lib"
        cd examples/ambs-lgp2-client
        autoreconf -vis
        ./configure --prefix="$PWD/_i"
        make
        ldd ambs-lgp2-client
        ./ambs-lgp2-client
        make install
        ldd _i/bin/ambs-lgp2-client
        _i/bin/ambs-lgp2-client
```

---

Thank you very much! This provides the part I was too lazy to look up. :-)

TBH, I just wanted see that I make actual commands work and look up the multiline thing later.